.github/workflows/continuous-integration.yml

```diff
@@ -73,9 +71,11 @@ jobs:
       - name: "Run tests"
         run: "composer exec phpunit -- --verbose"
 
-      - name: "Run tests with rollbar"
+      - name: "Run tests with rollbar, elastica, gelf"
         run: |
-          composer require psr/log:'^1.1|^2' --no-update
-          composer require rollbar/rollbar:^1.3 --no-update
+          composer require --no-update psr/log:'^1.1|^2|^3'
+          composer require --no-update --dev rollbar/rollbar:^1.3
+          composer require --no-update --dev 'ruflin/elastica:>=0.90,<8'
+          composer require --no-update --dev graylog2/gelf-php:^1.4.2
```

Comment:
instead of having to sync those constraints with the `require-dev` in the package.json, it would make more sense to rely on jobs using the unaltered composer.json IMO.

Reply:
So, I would basically swap the jobs: first run the tests with all deps and then just remove those that block psr/log 3?

Reply:
Done.